starter-project-web-react/cypress/e2e/auth.spec.cy.ts

```diff
@@ -0,0 +1,34 @@
+describe('Authentication', ()=>{
+
+    it("should login user route to home", ()=>{
+        cy.visit('/auth/login')
+    
+        cy.get('[name="email"]').type(Cypress.env('TEST_EMAIL'))
```

Comment:
I am not entirely sure, but is using the "name" attribute the ideal way to find components, or is maybe adding cypress-specific attributes an option or sth? I can't remember clearly so if "name" is the best, stick with it, but otherwise it might be better to research more.